Source/Scene/GltfLoader.js

```diff
@@ -1443,14 +1403,78 @@ function loadNodes(loader, gltf, supportedImageFormats, frameState) {
     }
   }
 
+  return nodes;
+}
+
+function loadSkin(loader, gltf, gltfSkin, nodes) {
+  const skin = new Skin();
+
+  const jointIds = gltfSkin.joints;
+  const jointsLength = jointIds.length;
+  const joints = new Array(jointsLength);
+  for (let i = 0; i < jointsLength; ++i) {
+    joints[i] = nodes[jointIds[i]];
+  }
+  skin.joints = joints;
+
+  const inverseBindMatricesAccessorId = gltfSkin.inverseBindMatrices;
+  if (defined(inverseBindMatricesAccessorId)) {
+    const accessor = gltf.accessors[inverseBindMatricesAccessorId];
+    const bufferViewId = accessor.bufferView;
+    if (defined(bufferViewId)) {
```

Comment:
Looking at this code (and I see this is from before this PR), what should happen if this inner if statement is false? According to [the spec](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#_accessor_bufferview) it looks like an accessor is initialized to zeros unless it's a sparse accessor. 

I know sparse accessors are beyond the scope of this PR, but should this be handled like the else clause below?

Reply:
Yeah, let's handle it like the else clause for now.